api/conf/conf.yaml

```diff
@@ -80,6 +80,17 @@ authentication:
     - username: user
       password: user
 
+oidc:
+  secret:
+    secret
+  expire_time: 3600
+  apps:
+    - app_name: authing
+      client_id:
+      client_secret:
+      scope: oidc
+      redirect_uri: /authing/callback
```

Comment:
`redirect_uri` shouldn't contain the  `authing` config.
Maybe `oidc/callback`.

Reply:
ok